server/config/connection.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+mongoose.connect(
+  process.env.MONGODB_URI || 'mongodb://localhost:27017/parkaddventure',
+  {
+    useNewUrlParser: true, 
+    useUnifiedTopology: true,
+  }
```

Comment:
Looking at the other setups from the modules, I have these properties useCreateIndex: true,
  useFindAndModify: false, in addition to the other two you have.  I am not sure what those do/if we need them. ` useNewUrlParser: true,
  useUnifiedTopology: true,
  useCreateIndex: true,
  useFindAndModify: false,`